launcher_utils/advertise.go

```diff
@@ -54,11 +54,14 @@ type directorResponse struct {
 
 func doAdvertise(ctx context.Context, servers []server_structs.XRootDServer) {
 	log.Debugf("About to advertise %d XRootD servers", len(servers))
+	start := time.Now()
 	err := Advertise(ctx, servers)
+	duration := time.Since(start)
 	if err != nil {
-		log.Warningln("XRootD server advertise failed:", err)
+		log.Warningf("XRootD server advertise failed (duration %v): %s", duration.String(), err)
```

Comment:
I think the format specifiers might be backwards here (although I think this is probably fine, just not as idiomatic).